### DISCUSSION TITLE
Compound appearance in tissue

### DISCUSSION THREAD

**Discussion by DahlgrenDavid** [Open-Systems-Pharmacology/Forum#1851] [Questions & Problems]
Hi,

I want to create a model where an mRNA molecule appears at a constant rate and is degraded at a constant rate in a hepatocyte to create a steady state mRNA level which can then be altered. However I do not know how to create the steady state appearance of a molecule in a cell. 

How can that be done?

To get around the problem I tried creating a administration protocol where the mRNA molecule is infused into the hepatocytes, but infusion was only possible into venous blood.

Many thanks!
David

**Comment by Yuri05** [comment]
The best way is to model this in MoBi:
<img width="750" src="https://github.com/user-attachments/assets/ddf690dc-cdaa-4861-ab52-8d332c025c34">
<img width="400" src="https://github.com/user-attachments/assets/20987ee8-1758-45f8-b369-2e609b008d4c">

<details><summary><b>If you want to stay in PK-Sim (would not recommend it):</b></summary>
You could also "tweak" this in PK-Sim by adding an administration protocol into liver cells in combination with the zero order formulation for the synthesis and first order metabolism in liver cells for the degradation. However it would be pretty error prone and you will need to calculate the "dose", "administration and time" and the "clearance" so that at the end your constants are matched.

![grafik](https://github.com/user-attachments/assets/d38987d2-8d1f-4222-82c2-76a272724083)
![grafik](https://github.com/user-attachments/assets/078f484b-6c37-41c1-8481-01aeb39e838c)

</details>
